.eslintrc

```diff
@@ -45,10 +49,44 @@
         "@typescript-eslint/no-empty-function": "warn",
         "@typescript-eslint/no-unsafe-enum-comparison": "warn",
         "@typescript-eslint/restrict-template-expressions": "warn",
-        "@typescript-eslint/no-unused-vars": "error",
+        "@typescript-eslint/no-unused-vars": "warn",
         "@typescript-eslint/await-thenable": "warn",
         "@typescript-eslint/no-invalid-void-type": "warn",
         "@typescript-eslint/no-base-to-string": "warn",
-        "@typescript-eslint/restrict-plus-operands": "warn"
-    }
+        "@typescript-eslint/restrict-plus-operands": "warn",
+        "react/prop-types": "off"
+    },
+    "overrides": [
+        {
+            "files": [
+                "packages/webapp/**/*.tsx"
+            ],
+            "env": {
+                "browser": true,
+                "es6": true,
+                "node": false
+            },
+            "parserOptions": {
+                "ecmaVersion": 2018,
+                "sourceType": "module",
+                "ecmaFeatures": {
+                    "impliedStrict": true,
+                    "jsx": true
+                },
+                "project": "packages/webapp/tsconfig.json"
+            },
+            "rules": {
+                "no-console": "off",
+                "@typescript-eslint/no-misused-promises": [
+                    "warn",
+                    {
+                        "checksVoidReturn": {
+                            "arguments": false,
+                            "attributes": false
+                        }
```

Comment:
I ignored those errors because every `onClick={}` were complaining about async being used. I feel like too many warnings is not great and this is not really helping the codebase, wdyt?